components/sensor/LPS25H.c

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 PWrInSpace
+
+#include "LPS25H.h"
+
+esp_err_t LPS25HInit(LPS25H *lps, int sda, int scl, i2c_port_t portNum,
+                     uint8_t i2cAddress, i2c_config_t *conf) {
+  lps->port = portNum;
+  lps->addr = i2cAddress;
+  if ((void *)conf == NULL) {
```

Comment:
Po co rzutowanie?

Reply:
dobra w sumie już się nie pruje to usunąłem hehe